src/test/test-honor-first-shutdown.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Honor First Shutdown feature
+After=multi-user.target
+
+[Service]
+ExecStart=/test-honor-first-shutdown.sh
+ExecStop=pkill -9 test-honor-first-shutdown.sh
```

Comment:
```
ExecStop=kill -KILL $MAINPID
```

should just work, and not pull in pkill

Reply:
This pkill was the method I was using to cause termination of the loop inside the script and to cause the FailureAction=reboot emergency action during the shutdown. Otherwise the shutdown would just SIGTERM the script and no failure would be seen correct? I was thinking pkill might be a handy tool to have for the test environment. In any case, I will work on a different way so I won't have to pull in pkill.

Reply:
the line above is equivalent to the pkill. It uses the `$MAINPID` env var systemd passes to ExecStop= calls anyway, so the pkill is not necessary. The effect of the line I proposed and yours i the same: signal 9 (SIGKILL) is sent to the process invoked via ExecStart=

Reply:
Ah, got it. Will do. Thanks